packages/docbook_xml43.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Docbook_xml43 < Package
+  description 'document type definitions for verification of XML data files against the DocBook rule set'
+  compatibility 'all'
+  homepage 'http://www.docbook.org'
+  version '4.3'
+  source_url 'http://www.oasis-open.org/docbook/xml/4.3/docbook-xml-4.3.zip'
+  source_sha256 '23068a94ea6fd484b004c5a73ec36a66aa47ea8f0d6b62cc1695931f5c143464'
+
+  depends_on 'docbook'
+  depends_on 'xmlcatmgr'
+  depends_on 'docbook_xml'
+  depends_on 'docbook_xsl' # Requires the catalog.xml created within this package
+  
+  def self.prebuild
+    system "sed -i -e 's,<!-- .* -->,,g' #{CREW_PREFIX}/etc/xml/catalog.xml"
+    system "xmlcatmgr -c #{CREW_PREFIX}/etc/xml/docbook add public '-//OASIS//DTD DocBook XML V4.3//EN' '#{CREW_PREFIX}/share/xml/docbook/4.3/catalog.xml'"
```

Comment:
Does this command generate an output file?  If so, should this be an install step?

Reply:
It edits a file installed by `docbook_xml`
See #4315

Reply:
Should this command be executed in `def self.preinstall` instead?

Reply:
Yes, that would be a better place, as nothing is actually built.